src/System.Net.Http.Rtc/pkg/System.Net.Http.Rtc.pkgproj

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We need the lib for binary compatibility with previous PCLs which UAP supported.